### PR TITLE
Move djatoka_url in settings to driver.image.attributes.url

### PR DESCRIPTION
**Pull request by aaron-collier**
This moves driver specific settings into attribute values for the image and the info services. The image and info factories will now manage setting the attributes.

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 stacks:
   storage_root: /stacks
-  djatoka_url: 'http://localhost:8080/resolver'
   driver: djatoka
   djatoka:
     image: DjatokaImage
     info: DjatokaInfoService
+    resolver: 'http://localhost:8080/resolver'
```
Comment (cbeer) [inline config/settings.yml]: Is `resolver` the right, cross-implementation word? It seems more like a base url to me?

Comment (aaron-collier) [reply config/settings.yml]: Actually - that's a good question that I meant to ask this morning. Are these "resolvers" in RIIIF/Cantaloupe implementations, or should this just be `uri` and then (in this case) Djatoka is using that for it's "resolver"?

Comment (aaron-collier) [reply config/settings.yml]: Changed to URL.

File: config/settings.yml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 stacks:
   storage_root: /stacks
-  djatoka_url: 'http://localhost:8080/resolver'
   driver: djatoka
   djatoka:
     image: DjatokaImage
     info: DjatokaInfoService
+    url: 'http://localhost:8080/resolver'
```
Comment (jcoyne) [inline config/settings.yml]: This really seems like a property of DjatokaImage.   How would you feel about:
```
   djatoka:
     image:
       class: DjatokaImage
       attributes:
          url: 'http://localhost:8080/resolver'
     info:
       class: DjatokaInfoService

```

Comment (jcoyne) [reply config/settings.yml]: Then we can have the factory set all the attributes.

Comment (aaron-collier) [reply config/settings.yml]: WFM

**Pull request merged**